src/api/routes/address.ts

```diff
@@ -246,7 +226,7 @@ export function createAddressRouter(db: PgStore, chainId: ChainID): express.Rout
       } else {
         blockHeight = await getBlockHeight(untilBlock, req, res, next, db);
       }
-      const limit = parseTxQueryLimit(req.query.limit ?? 20);
+      const limit = getPagingQueryLimit('/address/:principal/transactions');
```

Comment:
Nit: would be a bit cleaner to use an `enum` or `const enum` for `pagedApiRoutes`, so that all the other references can avoid duplicating long strings and instead use something like:
```ts
getPagingQueryLimit(PagedApiRoutes.AddressPrincipalTransactions);
```

Reply:
I considered this. Utlimately my feeling is because routes don't translate well to sensible names, ie AddressPrincipalTransactions (You can imagine another being something like AddressPrincipalTransactionsID), they don't really add much value and can even add an element of misdrection when trying to understand the code. Here you can easily see you need to enter the route and what route limit you are fetching. I typed the route parameter so that it's not possible to accidently misspell the route.

Reply:
That sounds reasonable to me. Thanks @BLuEScioN.